src/main/java/net/countercraft/movecraft/combat/features/tracking/KillCredCommand.java

```diff
@@ -0,0 +1,39 @@
+package net.countercraft.movecraft.combat.features.tracking;
```

Comment:
Let's move this into a new sub-package of `tracking` called `commands` to mirror how the tracer commands are organized.